start_coupled.R

```diff
@@ -327,7 +327,6 @@ start_coupled <- function(path_remind, path_magpie, cfg_rem, cfg_mag, runname, m
         message(subsequentcommand)
         if (length(needfulldatagdx) > 0) {
           system(subsequentcommand)
-          Sys.sleep(10)
```

Comment:
I'm not quite sure if we should remove the sleep already. If people are running current REMIND develop with old snapshots with the old locking, this will make it more likely that things break, right?

Reply:
agreed. I undid it and added it our wishlist so I will not forget it: https://github.com/remindmodel/remind/issues/834